server/run-mysql.sh

```diff
@@ -1,3 +1,20 @@
-echo 'CREATE DATABASE mcm;' > /tmp/init.sql
```

Comment:
I don't think else should work without DB

Reply:
This is handled by the env variable `-e MYSQL_DATABASE=$DATABASE_SCHEMA`.

Please refer to the following for reference: https://hub.docker.com/_/mysql